packages/core/src/createActor.ts

```diff
@@ -11,6 +11,16 @@ import { reportUnhandledError } from './reportUnhandledError.ts';
 import { symbolObservable } from './symbolObservable.ts';
 import { AnyActorSystem, Clock, createSystem } from './system.ts';
 
+// those are needed to make JSDoc `@link` work properly
+import type {
+  fromObservable,
```

Comment:
We can't freely `{@link fromObservable}` in the JSDoc without that being available in the scope.